pkg/inspector/image-inspector.go

```diff
@@ -298,3 +329,19 @@ func getAuthConfigs(dockercfg, username, password_file string) (*docker.AuthConf
 
 	return imagePullAuths, nil
 }
+
+func (i *defaultImageInspector) scanImage(s openscap.Scanner) ([]byte, error) {
+	log.Printf("%s scanning %s", s.ScannerName(), i.opts.DstPath)
+	err := s.Scan(i.opts.DstPath, &i.meta.Image)
+	if err != nil {
+		err = fmt.Errorf("Unable to run %s: %v\n", s.ScannerName(), err)
+	} else {
+		scanReport, err := ioutil.ReadFile(s.ResultsFileName())
+		if err != nil {
+			err = fmt.Errorf("Unable to read %s result file: %v\n", s.ScannerName(), err)
+		} else {
+			return scanReport, nil
+		}
+	}
+	return []byte(""), err
+}
```

Comment:
Just a stylistic thing so you don't have to change this but I find that go reads easier if you just return early.  This method would look like:

```go
func (i *defaultImageInspector) scanImage(s openscap.Scanner) ([]byte, error) {
 	log.Printf("%s scanning %s", s.ScannerName(), i.opts.DstPath)
 	err := s.Scan(i.opts.DstPath, &i.meta.Image)
 	if err != nil {
 		return nil, fmt.Errorf("Unable to run %s: %v\n", s.ScannerName(), err)
 	} 
 	scanReport, err := ioutil.ReadFile(s.ResultsFileName())
 	if err != nil {
 		return nil, fmt.Errorf("Unable to read %s result file: %v\n", s.ScannerName(), err)
 	}
 	return scanReport, nil 	
 }
```